requirements/tox.txt

```diff
@@ -1,166 +1,161 @@
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
     #   google-api-core
 httplib2==0.22.0
     # via
-    #   -r requirements/../function_requirements/common-requirements.txt
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
     #   google-api-python-client
     #   google-auth-httplib2
-idna==3.4
+idna==3.10
     # via
-    #   -r requirements/../function_requirements/common-requirements.txt
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
     #   requests
-iniconfig==2.0.0
+iniconfig==2.1.0
     # via pytest
 jmespath==1.0.1
     # via
-    #   -r requirements/../function_requirements/common-requirements.txt
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
     #   boto3
     #   botocore
-mock==5.0.2
-    # via -r requirements/tox.in
+mock==5.2.0
+    # via -r tox.in
 oauthlib==3.2.2
     # via
-    #   -r requirements/../function_requirements/common-requirements.txt
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
     #   requests-oauthlib
-packaging==23.1
+packaging==25.0
     # via pytest
-pluggy==1.0.0
+pluggy==1.6.0
     # via pytest
-protobuf==4.23.2
+proto-plus==1.26.1
+    # via
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
+    #   google-api-core
+protobuf==6.31.0
     # via
-    #   -r requirements/../function_requirements/common-requirements.txt
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
     #   google-api-core
     #   googleapis-common-protos
-pyasn1==0.5.0
+    #   proto-plus
+pyasn1==0.6.1
     # via
-    #   -r requirements/../function_requirements/common-requirements.txt
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
     #   pyasn1-modules
     #   rsa
-pyasn1-modules==0.3.0
+pyasn1-modules==0.4.2
     # via
-    #   -r requirements/../function_requirements/common-requirements.txt
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
     #   google-auth
-pyjwt==2.7.0
-    # via -r requirements/../function_requirements/common-requirements.txt
-pyparsing==3.0.9
+pyjwt==2.10.1
+    # via -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
+pyparsing==3.2.3
     # via
-    #   -r requirements/../function_requirements/common-requirements.txt
+    #   -r /Users/ras300/Documents/opencast-16.x/zoom-ingester-oc-rute-16x/function_requirements/common-requirements.txt
```

Comment:
These are just annotation comments but it looks strange. I wonder why it's doing this.